src/python/WMCore/WebTools/RESTFormatter.py

```diff
@@ -75,7 +75,8 @@ def to_string(self, data):
         # testSupportedFormat (WMCore_t.WebTools_t.RESTFormat_t.RESTFormatTest)
         # fails with "ValueError: Page handlers MUST return bytes. 
         # Use tools.encode if you wish to return unicode."
-        return bytes(str(data), "utf-8")
+        # return bytes(str(data), "utf-8")
+        return str(data)
```

Comment:
Dario, how about a safe conversion to str like `decodeBytesToUnicode`. Or maybe we execute that only in PY3, and in py2 we do this `str(data)`?

Reply:
Yeah yeah i know. we can do all these things. My purpose here was just to check if and why that explicit (and with retrospect rather ugly) conversion to bytes was necessary for dmwm/WMCore. Since no unittest failed with this change, i would simply remove the conversion and live without. As you mentioned in a private chat, e should also wait for CRAB confirmation

Reply:
Ah sorry! you proposed a `how about a safe conversion to str` actually, replacing `str()` altogether. I do not think it is really feasible, what if `data` is not a string? I think the purpose of this method was stringyfying lists,dictionaries, sets, etc to a str, so i would keep this `str()`

Reply:
Dario, to avoid bad surprises, how about the following then:
```
if not isinstance(data, bytes):
    return str(data)
return decodeBytesToUnicode(data)
```

this would make the method consistent, as provided from your changes, and also avoid possible issues if we try to convert a bytes to string. If you agree, then this would likely close that other issue as well :-D

Reply:
ok! works for me! fix coming!